### ISSUE TITLE
[Quest] Becoming a Spellfire Tailor

### ISSUE THREAD

**Issue by Fiky995** [dalaranwow/dalaran-wow#4874]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
I went to Netherstorm and used the item, the cast completes and there is an Arcane explosion-like animation.
**Expected behaviour**: Tell us what should happen instead
An NPC (Nether Wraith) should spawn.
**Steps to reproduce the problem**:

1.  get a character with 350+ tailoring
2.  Go to Shattrath City to get the quest (it's a gnome)
3.  go to netherstorm and use the item

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

Quest: http://it.wowhead.com/quest=10832/becoming-a-spellfire-tailor

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

Item to use: http://www.wowhead.com/item=31742/nether-wraith-beacon#comments
Item needed for the quest, dropped by the mob that doesnt spawn: 
http://www.wowhead.com/item=31741/nether-wraith-essence

![wowscrnshot_091317_111352](https://user-images.githubusercontent.com/13967525/30369408-afe0ea48-9874-11e7-9101-9bfb44074c63.jpg)
![wowscrnshot_091317_111356](https://user-images.githubusercontent.com/13967525/30369437-d13c50e2-9874-11e7-9c6a-10690a82063f.jpg)

![wowscrnshot_091317_111400](https://user-images.githubusercontent.com/13967525/30369522-16db33c0-9875-11e7-9def-e6c85f395fc9.jpg)

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Crispybuilt** [comment]
#3426 Duplicate
- Close this one.

**Comment by Fiky995** [comment]
the other one is closed too but the quest is still not working...
